app/views/institutions/index.html.haml

```diff
@@ -37,11 +37,19 @@
 
   %div.small-12.large-7.columns#institutions_list
     - @institutions.each do |institution|
+      - dropdown_id = "hours-dropdown-#{institution.id}"
+      %div.f-dropdown.small{ id: dropdown_id, data: { dropdown: { content: true } } }
+        = getHours(institution.id)
       %div.institution.row
         %div.small-6.large-6.columns
-          %h3.row.small-12.large-12.institution-name= institution.name
-          %p.row.small-12.large-12.institution-distance= "Distance: " + getDistance(institution).to_s
-          %p.row.small-12.large-12.institution-hours= getHours(institution.id)
+          %h3.row.institution-name= institution.name
+          %div.details.row
+            %span.columns.small-6.large-12.institution-distance= "Distance: " + getDistance(institution).to_s
+            %div.columns.small-6.large-12
+              - if @hours_present
+                %a.custom-button{ data: { dropdown: dropdown_id }, aria: { expanded: "false", controls: dropdown_id } }= "Click here for hours"
+              - else
+                %span= t('hours.missing')
```

Comment:
These lines are what are going to cause the merge conflict most likely. (The reason why Github says it can't automatically merge.) In Daniels branch `L45` was changed so that it it included a link. Also `L53-L55` I believe were changed.

When merging, keep master for the institution name and `View More Details` link.